miden-crypto/src/dsa/ecdsa_k256/mod.rs

```diff
@@ -0,0 +1,311 @@
+//! ECDSA (Elliptic Curve Digital Signature Algorithm) signature implementation over secp256k1
+//! curve.
```

Comment:
Let's specify here that here we are using specifically Keccak as the hash function.

Also, maybe we should rename this into `ecdsa_k256_keccak` even now.

Reply:
Done